src/test/versionbits_tests.cpp

```diff
@@ -255,25 +255,6 @@ BOOST_AUTO_TEST_CASE(versionbits_test)
     }
 }
 
-BOOST_AUTO_TEST_CASE(versionbits_sanity)
```

Comment:
I think it would be better to keep the "check the params make sense" checks separate from the "check ComputeBlockVersion does the right thing" checks.

Reply:
the sanity checks are a requirement, so they need to be run as part of the same test case. Do you mean to split them up into a separate function? I think that'd be too verbose, but I am happy to take a diff if you happen to have one.

Reply:
Yeah, maybe you're right -- can't reliably test computeblockversion if some other deployment might start setting the bit in question as soon as it hits ACTIVE, even if they're not technically overlapping. No need to hold this up over it in any event.